x-pack/plugins/security_solution/public/management/routes.tsx

```diff
@@ -6,17 +6,21 @@
  */
 
 import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { MANAGEMENT_PATH } from '../../common/constants';
 import { ManagementContainer } from './pages';
 import { SecuritySubPluginRoutes } from '../app/types';
 import { CurrentLicense } from '../common/components/current_license';
+const queryClient = new QueryClient();
 
 /**
  * Returns the React Router Routes for the management area
  */
 const ManagementRoutes = () => (
   <CurrentLicense>
-    <ManagementContainer />
+    <QueryClientProvider client={queryClient}>
```

Comment:
@paul-tavares I put this queryClientProvider here that'll make react query work on all management pages and allow us to create our own settings for it. 

Another alternative is putting this on the security app level but I decided against it for now as we don't know if this will be more widely adopted.

Reply:
@paul-tavares I moved this to the security app level.